dapp_runner/log.py

```diff
@@ -0,0 +1,66 @@
+"""Python logging configuration for the `dapp-runner`."""
+import logging
+from typing import Optional
+import warnings
+
+from yapapi import __version__ as yapapi_version
+from yapapi.log import _YagnaDatetimeFormatter  # noqa
+
+logger = logging.getLogger(__name__)
+
+
+def enable_logger(
+    log_file: Optional[str] = None,
+    enable_warnings=False,
+    console_log_level=logging.INFO,
+    file_log_level=logging.DEBUG,
+    api_log_level=None,
+    format_: str = "[%(asctime)s %(levelname)s %(name)s] %(message)s",
+):
+    """Enable the logger.
+
+    By default, it outputs `INFO` level logs to stderr and `DEBUG` level logs
```

Comment:
I think you meant stdout, not stderr
```suggestion
    By default, it outputs `INFO` level logs to stdout and `DEBUG` level logs
```